gulpfile.js

```diff
@@ -42,4 +39,9 @@ gulp.task('watch', ['js'], function() {
 	gulp.watch(['js/**/*.js', '!js/public/**/*.js'], ['js']);
 });
 
-gulp.task('default', ['js']);
+gulp.task('karma', (done) => {
```

Comment:
and this syntax is causing the issues @jancborchardt is observing because older node versions don't support this